lib/package_builder/scripts/after_install.sh

```diff
@@ -16,4 +16,5 @@ mv -Tf /home/deploy/bops/current_<%= release %> /home/deploy/bops/current
 cd /home/deploy/bops/current && bundle install --without development test --deployment --quiet
 cd /home/deploy/bops/current && bundle exec rake db:migrate
 cd /home/deploy/bops/current && bundle exec rake assets:precompile
+if [ ${SERVER_TYPE} = "worker" ] ; then cd /home/deploy/bops/current && bundle exec whenever -w ; else echo not running whenever ; fi
```

Comment:
not part of this

Reply:
its part of the acceptance testing for this ticket though?

Reply:
Gotcha